.travis.yml

```diff
@@ -0,0 +1,5 @@
+language: objective-c
```

Comment:
I was just using the same format as TABSwiftLayout, happy to change it! I'm new to Travis CI